fiddler/manifest.json

```diff
@@ -0,0 +1,84 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "ee617671-508e-4bb3-ba25-8815b11a16aa",
+  "app_id": "fiddler",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Fiddler gives visibility and explainability for AI model performance",
+    "title": "Fiddler",
+    "media": [
+      {
+        "caption": "Fiddler Dashboard in DataDog",
+        "image_url": "images/fiddler-datadog.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Model Accuracy Chart",
+        "image_url": "images/accuracy-drift.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Model Analytics",
+        "image_url": "images/analytics.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Performing counter factual explanations",
+        "image_url": "images/counterfactual.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Data Drift Charts",
+        "image_url": "images/data-drift.png",
+        "media_type": "image"
+      },
+      {
+        "caption": "Model Explanations",
+        "image_url": "images/explanation.png",
+        "media_type": "image"
+      }
+    ],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::Mac OS",
+      "Supported OS::Windows",
+      "Category::Alerting",
+      "Category::Monitoring",
+      "Category::Metrics",
+      "Offering::Integration"
+    ]
+  },
+  "author": {
+    "name":"Danny Brock",
+    "homepage":"http://www.fiddler.ai",
+    "support_email": "danny@fiddler.ai",
+    "sales_email": "danny@fiddler.ai"
```

Comment:
This should be an alias not a specific persons email. Can you update this to reflect the support alias and sales alias?